src/vector/feature.rs

```diff
@@ -87,4 +150,12 @@ impl FieldValue {
             _ => panic!("not a RealValue")
         }
     }
+
+    /// Interpret the value as `i32`. Panics if the value is something else.
+    pub fn as_int(self) -> i32 {
```

Comment:
Should I also change the name of the two previous methods (`as_real` -> `to_real` and `as_string` `to_string` ?)